zulip_bots/zulip_bots/bots/game_of_fifteen/doc.md

```diff
@@ -0,0 +1,16 @@
+# game_of_fifteen bot
+
+Game of Fifteen bot is a bot designed to let you play a game of fifteen. To summon it, simply type `@game_of_fifteen`,
+like so:
+
+```
+@game_of_fifteen
+```
+
+Run this bot as described in [here](https://zulipchat.com/api/running-bots#running-a-bot).
+
+## Usage
+
+The goal of the game is to arrange all numbers from smallest to largest, starting with the grey question mark in the upper left corner, and then moving through each row until we hit the end.
+
```

Comment:
Move this paragraph to the top of the previous section. Also, replace `we hit the end` with `till one reaches the end`.